src/main/java/core/basesyntax/service/impl/CsvDataProcessingService.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.DataConverterService;
+import core.basesyntax.service.DataProcessingService;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class CsvDataProcessingService implements DataProcessingService {
+    private OperationStrategy operationStrategy;
+    private DataConverterService dataConvertingToObjects;
+
+    public CsvDataProcessingService(OperationStrategy operationStrategy,
+                                    DataConverterService dataConvertingToObjects) {
+        this.operationStrategy = operationStrategy;
+        this.dataConvertingToObjects = dataConvertingToObjects;
+    }
+
+    @Override
+    public void importDataToStorage() {
```

Comment:
this method should process data, not import it. Import means just taking data, but you are processing it
also, it should have a variable 
```suggestion
    public void processData(List<FruitTransaction> transactions) {
```